core/src/test/resources/HistoryServerExpectations/executor_list_json_expectation.json

```diff
@@ -4,11 +4,13 @@
   "rddBlocks" : 8,
   "memoryUsed" : 28000128,
   "diskUsed" : 0,
+  "totalCores" : 0,
```

Comment:
I updated that file according to the javadocs for HistoryServerSuite, the test that uses it. I would assume it's 0 since it's the driver and the driver has no cores assigned to run tasks.